tests/Core/Query/HelperTest.php

```diff
@@ -28,51 +28,51 @@ public function setUp()
     public function testRangeQueryInclusive()
     {
         $this->assertEquals(
-            'field:[1 TO 2]',
+            'field:["1" TO "2"]',
             $this->helper->rangeQuery('field', 1, 2)
         );
 
         $this->assertSame(
-            'store:[45,-94 TO 46,-93]',
+            'store:["45,-94" TO "46,-93"]',
             $this->helper->rangeQuery('store', '45,-94', '46,-93')
         );
     }
 
     public function testRangeQueryExclusive()
     {
         $this->assertSame(
-            'field:{1 TO 2}',
+            'field:{"1" TO "2"}',
             $this->helper->rangeQuery('field', 1, 2, false)
         );
 
         $this->assertSame(
-            'store:{45,-94 TO 46,-93}',
+            'store:{"45,-94" TO "46,-93"}',
             $this->helper->rangeQuery('store', '45,-94', '46,-93', false)
         );
     }
 
     public function testRangeQueryInclusiveNullValues()
     {
         $this->assertSame(
-            'field:[1 TO *]',
+            'field:["1" TO "*"]',
             $this->helper->rangeQuery('field', 1, null)
         );
 
         $this->assertSame(
-            'store:[* TO 46,-93]',
+            'store:["*" TO "46,-93"]',
             $this->helper->rangeQuery('store', null, '46,-93')
         );
     }
 
     public function testRangeQueryExclusiveNullValues()
     {
         $this->assertSame(
-            'field:{1 TO *}',
+            'field:{"1" TO "*"}',
             $this->helper->rangeQuery('field', 1, null, false)
         );
 
         $this->assertSame(
-            'store:{* TO 46,-93}',
+            'store:{"*" TO "46,-93"}',
```

Comment:
Hum: are we sure those are the expected resuts? the second argument `to "46,-93"]` would match a string, 
If the field is an integer, I guess it would not match any result. WDYT @mkalkbrenner?

Do we have any integration test for this scenario?